packages/blockchain-wallet-v4/src/redux/data/btc/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface FetchTransactionsAction {
```

Comment:
I would keep this type in the sagas file and export it, then join all the types coming from actions.ts, actionTypes.ts, reducers.ts etc... in this top level types.ts file.

Reply:
Agree 100%